packages/ui/src/components/multiselect/components/multiselect-input.tsx

```diff
@@ -58,7 +58,7 @@ const _MultiselectInput = <T,>(
         onKeyDown={onKeyDown}
         {...rest}
         className={classes(
-          'flex flex-row items-center justify-between rounded-xl border border-neutral-200 bg-white px-3 py-2 text-neutralGreen-900 shadow-xs ring-1 ring-[#D1D5DB] focus:outline-none focus:ring-indigo-500',
+          'flex flex-row items-center justify-between min-h-11 rounded-xl border border-secondary bg-white px-3 py-2 text-neutralGreen-900 shadow-xs ring-1 ring-[#D1D5DB] focus:outline-none focus:ring-indigo-500',
```

Comment:
we can stay with the neutral-200 to not duplicate the colors

Reply:
neutral-200 its #e5e5e5 (229,229,229)
on figma it's #ebecee (235,236,238)

Reply:
agreed